code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -39,7 +39,7 @@
 			message_admins("[ADMIN_LOOKUPFLW(usr)] has primed a [name] for detonation at [A.name][ADMIN_JMP(bombturf)].")
 			log_game("[key_name(usr)] has primed a [name] for detonation at [A.name] [COORD(bombturf)].")
 			to_chat(user, "<span class='warning'>You prime the [name]! [det_time / 10] second\s!</span>")
-			playsound(user.loc, 'sound/weapons/armbomb.ogg', 60, 1)
+			playsound(user.loc, arming_sound, 60, 1)

```

Comment:
Why is this being changed

Reply:
so i don't have to rewrite the entirety of prime() just so that the druggy duck squeaks instead of the normal arming sound